master_startup.py

```diff
@@ -205,20 +207,25 @@ def get_events_list(self):
             # Set full power.
             esc_pwm.ChangeDutyCycle(ESC_MAX_DUTY)
 
-            # TODO: What condition trips this state into LANDED?
-            # landed_time = datetime.datetime.now()
+            # If the acceleration magnitude is next to nothing following the freefall, set state to LANDED
+            if imu_data.get_acc_magnitude() < .1 * 9.81 && imu_data.get_acc_magnitude() > -.1 * 9.81 # Accounting random noise
```

Comment:
`&&` in C++ is `and` in Python. This could be written cleaner with `abs()`...

Reply:
Oh man, too many languages, recently... good catch.